### PR TITLE
Implements User

### PR DESCRIPTION
**Pull request by taliyada**
## 📝 Description

Implements User Client to get information about the user

### ✅ Checklist:

- [x] I followed the instructions indicated in [Contribution Guidelines](CONTRIBUTING.md)
- [x] Added the necessary tests for the code or functionality that I'm working on
- [x] I ran the tests and made sure they work
- [x] My branch coverage is at least 80%
- [x] I verified the style of my code matches the one in the project

## 🧰 How to reproduce

- Step by step of how to test, specially for bugs.
- Links of external docs.

## 📄 References

- Links do external documentation
- Diagrams
- Useful links

### CODE REVIEW

File: src/clients/user/get/types.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import type { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+
+export declare type UserGet = {
+  config: MercadoPagoConfig;
+};
+
+export declare type UserResponse = {
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts:7]: Tali olhando o php faltam muitas propriedades, veja:
https://github.com/mercadopago/sdk-php/blob/develop-v3/src/MercadoPago/Resources/User.php

Comment (taliyada) [reply src/clients/user/get/types.ts:7]: @eltinMeli na SDK de Java tem mapeados menos campos, qual dos dois a gente segue?
https://github.com/mercadopago/sdk-java/blob/master/src/main/java/com/mercadopago/resources/user/User.java

File: src/clients/user/get/types.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  credit: {
+    consumed: number;
+    credit_level_id: string;
+    rank: string;
+  };
+  context: {
+    ip_address: string;
+  };
+  registration_identifiers: string[];
+  countryId: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
  country_id: string;
```

File: src/clients/user/get/types.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    state: string | null;
+    zip_code: string | null;
+  };
+  phone: {
+    areaCode: string;
+    extension: string;
+    number: string;
+    verified: boolean;
+  };
+  alternativePhone: {
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
  alternative_phone: {
```

File: src/clients/user/get/types.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    extension: string;
+    number: string;
+  };
+  tags: string[];
+  userType?: string;
+  logo?: string;
+  points?: number;
+  siteId: string;
+  permalink?: string;
+  sellerExperience?: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
  seller_experience?: string;
```

File: src/clients/user/get/types.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  bill_data: {
+    acceptCreditNote?: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
    accept_credit_note?: string;
```

File: src/clients/user/get/types.ts [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+  };
+  sellerReputation: {
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
  seller_reputation: {
```

File: src/clients/user/get/types.ts [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+    number: string;
+    verified: boolean;
+  };
+  alternativePhone: {
+    areaCode: string;
+    extension: string;
+    number: string;
+  };
+  tags: string[];
+  userType?: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
  user_type?: string;
```

File: src/clients/user/get/types.ts [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    zip_code: string | null;
+  };
+  phone: {
+    areaCode: string;
+    extension: string;
+    number: string;
+    verified: boolean;
+  };
+  alternativePhone: {
+    areaCode: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
    area_code: string;
```

File: src/clients/user/get/types.ts [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+    type: string;
+  };
+  address: {
+    address: string | null;
+    city: string | null;
+    state: string | null;
+    zip_code: string | null;
+  };
+  phone: {
+    areaCode: string;
```
Comment (eltinMeli) [inline src/clients/user/get/types.ts]:
```suggestion
    area_code: string;
```

**Review by eltinMeli — APPROVED** [review]

File: src/clients/user/get/types.ts [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+  user_type?: string;
+  logo?: string;
+  points?: number;
+  site_id: string;
+  permalink?: string;
+  seller_experience?: string;
+  bill_data: {
+    accept_credit_note?: string;
+  };
+  seller_reputation: {
```
Comment (lucmkz) [inline src/clients/user/get/types.ts]: não vamos deixar acontecer essa identação aqui, vamos sempre criar as tipagens para cada atributo, pois elas podem ser reutilizadas em algum momento, isso vale para as outras ocorrencias tb

**Review by lucmkz — CHANGES_REQUESTED** [review]

File: src/clients/user/get/types.ts [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+
+export declare type BillData = {
+  accept_credit_note?: string;
+};
+
+export declare type SellerReputation = {
+  level_id: string | null;
+  power_seller_status: string | null;
+  transactions: {
+    canceled: number;
```
Comment (lucmkz) [inline src/clients/user/get/types.ts]: remover todas as indentações
sempre declarar o tipo de cada atributo, o meso vale para as outras ocorrências

**Review by lucmkz — CHANGES_REQUESTED** [review]

**Review by lucmkz — APPROVED** [review]

**Pull request merged**